src/robot/reporting/resultwriter.py

```diff
@@ -36,8 +36,8 @@ class ResultWriter(object):
     or enabled using ``settings`` or ``options`` passed to the
     :meth:`write_results` method. The latter is typically more convenient::
 
-        writer = ResultWriter(result)
-        writer.write_results(report='custom.html', log=None, xunit='xunit.xml')
+        _writer = ResultWriter(result)
+        _writer.write_results(report='custom.html', log=None, xunit='xunit.xml')
```

Comment:
This doesn't look relevant. Automatic refactoring by an IDE gone wrong?

Reply:
correct